code/modules/client/preferences.dm

```diff
@@ -19,6 +19,14 @@ datum/preferences
 	var/ui_style = "Midnight"
 	var/tgui_fancy = TRUE
 	var/tgui_lock = FALSE
+	var/enable_tips = TRUE
+	var/tip_delay = 500 //tip delay in milliseconds
+
+	///Do we show item hover outlines?
+	var/itemoutline_pref = TRUE
```

Comment:
if this feature is going to be added it should not be optional, people would turn it off to get a gameplay advantage so nobody can see where they're looking

Reply:
> if this feature is going to be added it should not be optional, people would turn it off to get a gameplay advantage so nobody can see where they're looking

reverted